VOXELPLA PLUS BLACK.xml.fdm_material

```diff
@@ -0,0 +1,252 @@
+      <setting key="retraction speed">60</setting>
+      <setting key="retraction amount">5</setting>
+    </machine>
+    <machine>
+      <machine_identifier manufacturer="Creality3D" product="creality_cr10" />
+      <setting key="heated bed temperature">50.0</setting>
+      <setting key="standby temperature">180.0</setting>
+      <setting key="print cooling">100</setting>
+      <setting key="retraction speed">60.0</setting>
+      <setting key="retraction amount">5.0</setting>
+    </machine>
+    <machine>
+      <machine_identifier manufacturer="CreawsomeMod" product="creality_base" />
+      <setting key="heated bed temperature">50</setting>
+      <setting key="standby temperature">180</setting>
+      <setting key="print cooling">100</setting>
+      <setting key="retraction speed">60</setting>
+      <setting key="retraction amount">5</setting>
+    </machine>
+    <machine>
+      <machine_identifier manufacturer="Cartesio bv" product="cartesio" />
+      <setting key="print temperature">190</setting>
+      <setting key="heated bed temperature">43</setting>
+      <setting key="standby temperature">160</setting>
+      <setting key="print cooling">40.0</setting>
+      <setting key="retraction speed">40</setting>
+      <hotend id="0.8mm thermoplastic extruder">
+        <setting key="hardware compatible">yes</setting>
+        <setting key="retraction amount">1.5</setting>
+      </hotend>
+      <hotend id="0.4mm thermoplastic extruder">
+        <setting key="hardware compatible">yes</setting>
+        <setting key="retraction amount">1.0</setting>
+      </hotend>
+      <hotend id="0.25mm thermoplastic extruder">
+        <setting key="hardware compatible">yes</setting>
+        <setting key="retraction amount">1.0</setting>
+      </hotend>
+    </machine>
+    <machine>
+      <machine_identifier manufacturer="BIQU" product="biqu_b1_abl" />
+      <setting key="print temperature">180</setting>
+      <setting key="standby temperature">180</setting>
+      <setting key="print cooling">100</setting>
+    </machine>
+    <machine>
+      <machine_identifier manufacturer="BIQU" product="biqu_b1" />
+      <setting key="print temperature">180</setting>
+      <setting key="standby temperature">180</setting>
+      <setting key="print cooling">100</setting>
+    </machine>
+    <machine>
+      <machine_identifier manufacturer="Kati Hal ARGE" product="alyanx3dp" />
+      <setting key="standby temperature">180</setting>
+      <setting key="print cooling">100</setting>
+      <setting key="retraction speed">40</setting>
+      <setting key="retraction amount">1.5</setting>
+    </machine>
+    <machine>
+      <machine_identifier manufacturer="Kati Hal ARGE" product="alya3dp" />
+      <setting key="standby temperature">180</setting>
+      <setting key="print cooling">100</setting>
+      <setting key="retraction speed">40</setting>
+      <setting key="retraction amount">1.5</setting>
+    </machine>

```

Comment:
You do not have to repeat every printer definition if the settings are the same. 

If you check:
https://github.com/Ultimaker/fdm_materials/blob/master/redd_pla.xml.fdm_material
You can see that they removed the machine definitions and it's still available for all 1.75 printers in Cura. 

If you know the specific settings of each printer and they are indeed different, you should keep them in.